packages/genesys-spark-components/src/components/stable/gux-table/example.html

```diff
@@ -352,16 +352,20 @@ <h2>Data table with resizable columns</h2>
   <table slot="data">
     <thead>
       <tr>
-        <th data-column-name="first-name">First name</th>
-        <th data-column-name="last-name">Last name</th>
+        <th class="gux-sticky-cell" data-column-name="first-name">
```

Comment:
Should these have the style classes as well or where they just for testing and should be removed?

Reply:
Ye, you are correct I have now removed them they were for testing.